packages/sidecar/src/cli/tail.ts

```diff
@@ -59,7 +53,7 @@ export default async function tail({ port, cmdArgs, basePath, filesToServe }: CL
     : new Set([...eventTypes.flatMap(type => NAME_TO_TYPE_MAPPING[type] || [])]);
   const onEnvelope: (envelope: ParsedEnvelope["envelope"]) => void = envelope => {
     if (envelope[1].some(([header]) => header.type && types.has(header.type))) {
-      displayEnvelope(envelope);
+      displayEnvelope(envelope, format as FormatterType);
```

Comment:
Protip: we can have a "isFormatter" function with return type "is FormatterType", validate it before passing here and ensure the type?